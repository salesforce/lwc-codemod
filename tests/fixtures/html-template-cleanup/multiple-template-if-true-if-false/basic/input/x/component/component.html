<template>
    <section if:true={componentVisible} if:false={componentVisible}>
        <span>Show me the money!</span>
    </section>
</template>