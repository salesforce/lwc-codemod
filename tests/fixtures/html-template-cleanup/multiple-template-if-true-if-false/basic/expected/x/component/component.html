<template>
    <section if:true={componentVisible}>
        <span>Show me the money!</span>
    </section>
</template>